custom/icds/messaging/indicators.py

```diff
@@ -108,6 +127,37 @@ def location_names_from_user_id(self, user_ids):
         return {self.awc_locations[l] for l in loc_ids}
 
 
+class AWWAggregatePerformanceIndicator(AWWIndicator):
+    template = 'ls_aggregate_performance.txt'
+
+    def get_value_from_fixture(self, fixture, attribute):
+        xpath = './rows/row[@is_total_row="False"]'
+        rows = fixture.findall(xpath)
+        for row in rows:
+            owner_id = row.find('./column[@id="owner_id"]')
+            if owner_id.text == self.user.sql_location.name:
```

Comment:
i see, i think if that's going to be difficult it's probably also fine if we just fail hard if we can't find it. right now it would return None if none match